src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -228,6 +236,23 @@ export const GlobalProvider = memo(
             return () => clearTimeout(timerId);
         }, [nodeChanges, edgeChanges, manualOutputTypes, functionDefinitions]);
 
+        const [outputDataMap, setOutputDataMap] =
+            useState<ReadonlyMap<string, OutputData>>(EMPTY_MAP);
+
+        // useEffect(() => {
+        //     const timerId = setTimeout(() => {
+        //         sessionStorage.setItem('cachedOutputData', JSON.stringify([...outputDataMap]));
+        //     }, 500);
+        //     return () => clearTimeout(timerId);
+        // }, [outputDataMap]);
+        // useEffect(() => {
+        //     const cachedOutputData = getSessionStorageOrDefault<ReadonlyMap<string, OutputData>>(
+        //         'cachedOutputData',
+        //         EMPTY_MAP
+        //     );
+        //     setOutputDataMap(new Map([...cachedOutputData]));
+        // }, [setOutputDataMap]);
```

Comment:
Would it be a good idea to cache the outputData on refresh?

Reply:
IMO, no.